src/zenml/zen_stores/zen_store_interface.py

```diff
@@ -439,6 +431,19 @@ def get_user(self, user_name_or_id: Union[str, UUID]) -> UserResponseModel:
             KeyError: If no user with the given name or ID exists.
         """
 
+    @abstractmethod
+    def get_myself(
+        self, user_name_or_id: Optional[Union[str, UUID]] = None
```

Comment:
Without reviewing this in detail, I have two questions regarding this method:
1. I think safoine for the token inside docker image recently implemented a `current_user` call, is this now duplicated?
2. Why can I pass arguments like the name or ID to this method, doesn't that defeat the purpose of returning "me"?

Reply:
2. Answer see above, how would the SQLZenStore know what the active user is. We have that information on the endpoint in the auth_context, but the SQLZenStore does not and should not have this information

Reply:
I'm definitely open to suggestions though @fa9r @schustmi @safoinme

Reply:
Ah I see, I think my confusion came mostly from the naming of the method then. If this just gives me any arbitrary logged in used, can we rename it to reflect that? `get_logged_in_user`, `get_full_user`, ...?

Reply:
Actually `get_logged_in_user` isn't any better, I think it needs to somehow describe the difference between users and "full" users

Reply:
I have now applied a suggestion I got from safoine, I removed the get_myself() method and moved everything onto get user

if you don't explicitely set the user_name_or_id you get the active user

if you get the active user, you can then opt to get the private information as well (this happens implicitely)

Reply:
Oh I like that, looks great now!